backend/app/api/api_v1/endpoints/expectation.py

```diff
@@ -41,52 +40,22 @@ def list_supported_expectations():
     return JSONResponse(status_code=status.HTTP_200_OK, content=content)
 
 
-@router.put("/{expectation_id}/enable", response_model=Expectation)
+@router.put("/{expectation_id}/enable")
```

Comment:
The problem is the same as stated above for `Datasource`: since we actually have objects inheriting from `Expectation`, we'll lose all their specific fields if we set the `response_model`, the output will be "down-casted" to a base `Expectation`.